src/app/components/nav/header-static/header-static.component.ts

```diff
@@ -4,7 +4,6 @@ import { AuthService } from '../../../services/auth.service';
 import { RouterModule, Router, ActivatedRoute, NavigationEnd } from '@angular/router';
 import { DOCUMENT } from '@angular/common';
 import {ApiService} from '../../../services/api.service';
-import {el} from '@angular/platform-browser/testing/src/browser_util';
```

Comment:
Please revert this change here @drepram

Reply:
This caused an error since the module was not found. I'll revert it and see if it doesn't cause the build to fail.